File-Formats/Presentation/Open-and-Save-PowerPoint-Presentation-in-WinUI.md

```diff
@@ -0,0 +1,331 @@
+
+Step 5: Add a new button to the **MainPage.xaml** as shown below.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+<Page
+    x:Class="Read_and_edit_PowerPoint_presentation.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Read_and_edit_PowerPoint_presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+        <Button x:Name="button" Click="OpenAndSavePresentation">Open and Save Presentation</Button>
+    </StackPanel>
+</Page>
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 6: Include the following namespaces in the **MainPage.xaml.cs** file.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.Presentation;
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 7: Add a new action method **OpenAndSavePresentation** in MainPage.xaml.cs and include the below code snippet to **open an existing PowerPoint Presentation in WinUI UWP app**.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+//Open an existing PowerPoint presentation.
+IPresentation pptxDoc = Presentation.Open(assembly.GetManifestResourceStream("Read_and_edit_PowerPoint_presentation.Assets.Template.pptx"));
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 8: Add below code snippet demonstrates accessing a shape from a slide and changing the text within it.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Get the first slide from the PowerPoint presentation.
+ISlide slide = pptxDoc.Slides[0];
+//Gets the first shape of the slide.
+IShape shape = slide.Shapes[0] as IShape;
+//Modify the text of the shape.
+if (shape.TextBody.Text == "Company History")
+    shape.TextBody.Text = "Company Profile";
+
+{% endhighlight %}
+{% endtabs %}
+
+Step 9: Add below code example to **save the PowerPoint Presentation in WinUI UWP app**.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Save the Presentation files to MemoryStream.
+MemoryStream stream = new MemoryStream();
+pptxDoc.Save(stream);
+//Save the stream as a Presentation file in the local machine.
+Save(stream);
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Read-and-save-PowerPoint-presentation/Open-and-save-PowerPoint/WinUI/WinUI-UWP-app).
+
+By executing the program, you will get the **PowerPoint document** as follows.
+
+![WinUI UWP app output PowerPoint document](Workingwith_Core/Open-and-Save-output-image.png)
+
+### Save Presentation file in WinUI UWP app
```

Comment:
Change the step by referring this https://help.syncfusion.com/file-formats/docio/open-and-save-word-document-in-winui#winui-uwp-app

Reply:
updated